packages/itwin/measure-tools/src/test/measure-tools/_Setup.test.ts

```diff
@@ -4,16 +4,27 @@
 *--------------------------------------------------------------------------------------------*/
 import { afterAll, beforeAll } from "vitest";
 
-import { NoRenderApp } from "@itwin/core-frontend";
+import { IModelApp, NoRenderApp } from "@itwin/core-frontend";
 import { MeasureTools } from "../../MeasureTools.js";
 import { TestUtils } from "../TestUtils.js";
 import { EmptyLocalization } from "@itwin/core-common";
+import { SchemaContext, SchemaUnitProvider } from "@itwin/ecschema-metadata";
+import { SchemaXmlFileLocater } from "@itwin/ecschema-locaters";
+
+import * as path from "path";
 
 // Before all tests, initialize any global services
 beforeAll(async () => {
   const localization = new EmptyLocalization();
   await NoRenderApp.startup({ localization });
   await MeasureTools.startup({ localization });
+
+  const schemaContext = new SchemaContext();
+  const unitSchemaFilePath = path.join(__dirname, "..", "..", "..", "node_modules", "@bentley", "units-schema");
```

Comment:
why not use `require.resolve`?

Reply:
fair point, changed to use require.resolve